### DISCUSSION TITLE
User Flow when added to Invite-only batches

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Ed/Community#572] [Sunbird ED configuration discussion] [answered]
Hello All,

I need help understanding the user flow if the user is added to an invite-only batch.

Following are my questions :

1. How does a user whether mentor or participant know if they are added to the batch?
2. What medium is used to tell users if they are added? Ex: SMS / Email?
3. When the user is added to the batch does that mean that the user is directly enrolled to the course batch when added?

Please guide me here. Looking for more input.

cc : @coolbung @komalm @Renuka-S @vinod303 @paritshivani

**Comment by ashokreddy1208** [comment, marked as answer]
@Xitija 

1. Once the mentor or participant is added to a batch, Added mentors and Participants will receive Email and notification (bell icon) on application
![image](https://github.com/Sunbird-Ed/Community/assets/58033030/1d9ab668-a0af-4372-ba3c-aeb836329a59)
2. Once user is added to a batch email notification will be sent.
3. Yes, the relevant course will be added to 'my course' section by default.

cc : @coolbung @komalm @Renuka-S @vinod303 @paritshivani @rajeevsathish @Krishnaj20
